shared/toks-components/src/components/ImageViewer/index.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import { ComponentProps, useState } from 'react';
+import PhotoAlbum from 'react-photo-album';
+import Lightbox from 'yet-another-react-lightbox';
+
+interface Props extends ComponentProps<typeof Lightbox> {
+  photos: Array<{ src: string }>;
+}
+
+export function ImageViewer({ photos }: Props) {
+  const [index, setIndex] = useState(-1);
+  // photoAlbum 고정 사이즈
+  const photoAlbumSize = 188;
+
+  const albumPhotos = photos.map(photo => ({
+    src: photo.src,
+    width: photoAlbumSize,
+    height: photoAlbumSize,
+    key: photo.src,
+  }));
+
+  return (
+    <PhotoAlbumWrapper>
+      <PhotoAlbum layout="rows" photos={albumPhotos} onClick={({ index }) => setIndex(index)} />
+      <Lightbox open={index >= 0} index={index} close={() => setIndex(-1)} slides={photos} />
+    </PhotoAlbumWrapper>
+  );
+}
+
+const PhotoAlbumWrapper = styled.div`
+  .react-photo-album--photo {
+    width: 188px !important;
+    border-radius: 8px;
+    object-fit: cover !important;
+    :first-child {
+      margin-left: 0px;
+    }
+    margin-left: 16px;
```

Comment:
```suggestion
    :nth-of-type(n + 2) {
      margin-left: 16px;
    }
```
이렇게도 바꿔볼 수 있을 것 같습니다!